tools/RecyclerChecker/RecyclerChecker.cpp

```diff
@@ -1,3 +1,8 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
 //===- RecyclerChecker.cpp ---------------------------------------------===//
```

Comment:
@digitalinfinity I'll remove the old license then. How about the 2 `modules/Find???.cmake` files? Do they need any license copy, or should we insert our license? (I'm currently excluding them from Jenkins.)